unstable/lens/string.rkt

```diff
@@ -0,0 +1,43 @@
+#lang racket/base
+
+(provide string-ref-lens
+         string-pluck-lens
+         )
+
+(require fancy-app
+         lens/base/main
+         "join.rkt"
+         )
+(module+ test
+  (require rackunit))
+
+(define (string-ref-lens i)
+  (make-lens
+   (string-ref _ i)
+   (string-set _ i _)))
```

Comment:
Yes, but since isomorphisms aren't provided right now, I thought it would be better to implement it like this for now, but change it once isomorphisms are added.